Description: Remove some gems from bundle - will use system libraries.
Author: Alexander Golovko <alexandro@ankalagon.ru>
Last-Update: 2013-03-02

diff --git a/Gemfile.lock b/Gemfile.lock
index d8be14b..2709cfb 100644
--- a/Gemfile.lock
+++ b/Gemfile.lock
@@ -159,7 +159,7 @@ GEM
       mail (~> 2.2)
     erubis (2.7.0)
     escape_utils (0.2.4)
-    eventmachine (1.0.0)
+    eventmachine (0.12.11)
     execjs (1.4.0)
       multi_json (~> 1.0)
     facter (1.6.17)
